docker/Dockerfile

```diff
@@ -27,20 +27,13 @@ RUN addgroup holochain -g 868 \
       GOBIN=/home/holochain/bin
   ENV PATH=$GOPATH/bin:/usr/local/go/bin:$GOBIN:$PATH
 
-  RUN go get -v -u github.com/whyrusleeping/gx
-  RUN go get -v -d github.com/metacurrency/holochain
-
-  WORKDIR /work/golang/src/github.com/metacurrency/holochain
-  RUN make deps
-
 WORKDIR /work/golang/src/github.com/metacurrency/holochain
 
 # make our development files available to hc, on top of the cached image "so far" from above
   ADD . /work/holochain
 
   # this rm line works well because it stops weird artifacts coming over from github.com/metacurrency/holochain::master
-  RUN rm -rf /work/golang/src/github.com/metacurrency/holochain/* \
```

Comment:
I think I had this because I was worried the make deps will fetch the "released" version to the directory so you could get stuff from two places. Probably not a big problem in any case.